### DISCUSSION TITLE
Better name for a `module_add` argument

### DISCUSSION THREAD

**Discussion by gogonzo** [insightsengineering/teal.slice#356] [Polls]
Linked to https://github.com/insightsengineering/teal.slice/issues/355

In `filter_settings` we introduced an argument called `module_add` which: 
https://github.com/insightsengineering/teal.slice/blob/0e79b9b79239caf8eef20f3527f3ac45f74db583/R/teal_slice.R#L79
This means when `module_add = FALSE` then `Add Filter Variables` in filter-panel won't be visible in whole application. Technically, user won't be able to add a new filter from a column. 

<img width="1509" alt="Skärmavbild 2023-06-22 kl  10 04 49" src="https://github.com/insightsengineering/teal.slice/assets/6959016/00a3e392-bc2d-4529-b18f-e5a34d71a9c8">

We debate about better name for this argument. 
@lcd2yyz @chlebowa @m7pr @kartikeyakirar Please vote, if you have other ideas please edit my poll

**Comment by m7pr** [comment]
I was considering `allow_more` and `allow_add`

**Comment by gogonzo** [comment]
`allow_add` it is
